scriptmodules/supplementary/moonlight-embedded.sh

```diff
@@ -0,0 +1,139 @@
+}
+
+function sources_moonlight-embedded() {
+    gitPullOrClone "$md_build/installer" "https://github.com/Nyaran/RetropieMoonlightInstaller.git"
+    gitPullOrClone "$md_build/moonlight" "https://github.com/irtimmer/moonlight-embedded.git"
+}
+
+# Compile CMake 3.1+ until is not available on Raspbian repositories
+function build_cmake() {
+    wget https://cmake.org/files/v3.6/cmake-3.6.2.tar.gz
+    tar -xvf cmake-3.6.2.tar.gz
+    rm cmake-3.6.2.tar.gz
+    pushd cmake-3.6.2/
+    ./bootstrap
+    make
+    popd
+}
+
+function build_moonlight-embedded() {
+    build_cmake
+
+    pushd moonlight
+    git submodule update --init
+    mkdir build
+    cd build/
+
+    ../../cmake-3.6.2/bin/cmake ../
+    make
+    popd
+}
+
+function install_moonlight-embedded() {
+    # Install moonlight-embedded simple theme
+    mkdir -p /etc/emulationstation/themes/simple/moonlight-embedded/art
+    cp -v "$md_build/installer/themes/simple/theme.xml" /etc/emulationstation/themes/simple/moonlight-embedded/theme.xml
+    cp -v "$md_build/installer/themes/simple/art/icon.png" /etc/emulationstation/themes/simple/moonlight-embedded/art/icon.png
+    cp -v "$md_build/installer/themes/simple/art/logo.png" /etc/emulationstation/themes/simple/moonlight-embedded/art/logo.png
+    cp -v "$md_build/installer/themes/simple/art/background.png" /etc/emulationstation/themes/simple/moonlight-embedded/art/background.png
+
+    # Install moonlight-embedded carbon theme
+    mkdir -p /etc/emulationstation/themes/carbon/moonlight-embedded/art
+    cp -v "$md_build/installer/themes/carbon/theme.xml" /etc/emulationstation/themes/carbon/moonlight-embedded/theme.xml
+    cp -v "$md_build/installer/themes/carbon/art/system.svg" /etc/emulationstation/themes/carbon/moonlight-embedded/art/system.svg
+    cp -v "$md_build/installer/themes/carbon/art/controller.svg" /etc/emulationstation/themes/carbon/moonlight-embedded/art/controller.svg
+
+    md_ret_files=(
+        'moonlight/build/moonlight'
+        'moonlight/build/libmoonlight-pi.so'
+        'moonlight/build/libgamestream/libgamestream.so'
+        'moonlight/build/libgamestream/libgamestream.so.0'
+        'moonlight/build/libgamestream/libgamestream.so.2.2.2'
+        'moonlight/build/libgamestream/libmoonlight-common.so'
+        'moonlight/build/libgamestream/libmoonlight-common.so.0'
+        'moonlight/build/libgamestream/libmoonlight-common.so.2.2.2'
+        'moonlight/build/docs'
+    )
+}
+
+function configure_moonlight-embedded() {
+    # Create romdir moonlight embedded
+    mkRomDir "moonlight-embedded"
+
+    # Add System to es_system.cfg
+    addSystem 1 "$md_id" "moonlight-embedded" "%ROM%" "$rp_module_desc" ".sh .SH"
+
+    [[ "$md_mode" == "remove" ]] && return
+
+    # Create moonlight embedded config script
+    cat > "$romdir/moonlight-embedded/moonlight-embedded_config.sh" << _EOF_
+#!/bin/bash
+sudo $scriptdir/retropie_packages.sh moonlight-embedded configure
+_EOF_
+
+    gui_moonlight-embedded

```

Comment:
no gui stuff should be included in the configure_ function - users can choose to configure after installing.